pages/project/[slug].js

```diff
@@ -7,27 +7,54 @@ export default function Tasks() {
   const router = useRouter();
   const { slug } = router.query;
 
-  const projectIndex = projects.findIndex((project) => project.slug === slug);
-
-  const project = projects[projectIndex];
+  const project = projects.find((project) => project.slug === slug);
 
   if (!project) {
-    return null;
+    return <p>404 page</p>;
```

Comment:
okay, that is good for now, consider creating a dedicated page at some later point in time